services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/mapper/DocumentTableSchema.java

```diff
@@ -109,12 +112,12 @@ public Map<String, AttributeValue> itemToMap(EnhancedDocument item, boolean igno
     }
 
     private List<AttributeConverterProvider> mergeAttributeConverterProviders(EnhancedDocument item) {
-        List<AttributeConverterProvider> providers = new ArrayList<>();
+        Set<AttributeConverterProvider> providers = new LinkedHashSet<>();
```

Comment:
Yes , added this because of duplicate providers can get added from TableSchema and Document.
Currently functionally there is no change , only thing is since we add DefaultProviders in table , if user mentions it again in Document then it used to end up with two providers. The ChainProvider will necessarily loop through the duplicates.